.github/workflows/ci-on-push.yaml

```diff
@@ -28,4 +28,5 @@ jobs:
       commit-hash: ${{ github.event.pull_request.merge_commit_sha }}
```

Comment:
Do we want to change this back to `commit-hash: ${{ github.event.pull_request.head.sha }}` now we are rebasing ourselves?

Reply:
We do! :-)

Reply:
Fixed!